modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm

```diff
@@ -65,7 +65,7 @@
 	var/static/items_inside = list(
 		/obj/item/milking_machine/constructionkit = 1,
 		/obj/item/reagent_containers/glass/beaker = 1,
-		/obj/item/stock_parts/cell = 1,
+		/obj/item/stock_parts/cell/upgraded = 1, //please, let it be. 1 lvl Cell makes machine almost useless, charge lasts only for 2 minutes.
 		/obj/item/screwdriver = 1,
 		/obj/item/multitool = 1,
 		/obj/item/wrench = 1)
```

Comment:
It shouldn't have any of these things. Just a vastly cheaper way to get a multitool, not to mention the other tools that come with it